lib/evmone/tracing.cpp

```diff
@@ -82,15 +83,16 @@ class InstructionTracer : public Tracer
     const char* const* m_opcode_names = nullptr;
     std::ostream& m_out;  ///< Output stream.
 
-    void output_stack(const Stack& stack)
+    void output_stack(const intx::uint256* stack_top, int stack_height)
     {
-        const auto top = stack.size() - 1;
         m_out << R"(,"stack":[)";
-        for (int i = top; i >= 0; --i)
+        const auto stack_end = stack_top + 1;
+        const auto stack_begin = stack_end - stack_height;
+        for (auto it = stack_begin; it != stack_end; ++it)
```

Comment:
Did the order change here?

Reply:
The order is left unchanged. Previously we used `[i]` in reverse order to print bottom-up. Now we first calculate bottom pointer so the loop looks more normal.